bindings/sendgrid/sendgrid.go

```diff
@@ -0,0 +1,131 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package sendgrid
```

Comment:
Do we think sendgrid should be under the twilio namespace like twilio.sms?

Reply:
I considered this, it was only when I was done I realised Twilio now owns SendGrid 
AFAIK, it doesn't affect the package name, just the registration in `daprd/main.go`

Reply:
Yeh I didn't know where else to put it :P - but to be consistent we'd want to move the directory structure to `twilio/sms` and `twilio/sendgrid` - this would change the import path of both in dapr i.e. `"github.com/dapr/components-contrib/bindings/twilio"` would become `"github.com/dapr/components-contrib/bindings/twilio/sms"` at https://github.com/dapr/dapr/blob/76b06ea574d071f95e879b77460ad5c2d697b893/cmd/daprd/main.go#L93.

Reply:
Don't mind either way.
I did find it slightly confusing as a first time contributor that the names for registered components was sometimes different from the package name & dir structure. But a quick scan of main.go and all became clear

Reply:
I agree, it should go under `twillio/sendgrid`

Reply:
Done :)